lib/KRadioButton.vue

```diff
@@ -13,7 +13,7 @@
           type="radio"
           class="k-radio-button-input"
           :checked="isChecked"
-          :value="value"
+          :buttonValue="buttonValue"
```

Comment:
The changes look good and the radio button works as intended, thank you for updating this! Regarding this HTML `input` element, since it does not have an attribute called `buttonValue`, my only suggestion would be to set this line as `:value = buttonValue`

Reply:
Changes made.